### PR TITLE
adding in launch party picture; removing duplicate index.html

### PR DESCRIPTION
**Pull request by slackish**

### CODE REVIEW

File: site/index.md [hunk 1]
```diff
@@ -9,9 +9,11 @@ layout: home
 #  a makerspace in Roanoke, VA.
 ---
 
+### MAKE Roanoke is launching a Kickstarter campaign to open a Makerspace in Roanoke, VA
+
 MAKE Roanoke has already raised nearly $10,000 from individual contributions, membership dues, class fees and more. We’re currently negotiating a lease for a location in the heart of Roanoke. We've also received some generous donations of equipment, but we need YOUR HELP to cross the finish line!
 
-![](/assets/images/Kickstarter.png)
+![https://meetup.com/make-roanoke](/assets/images/20241017_launch_party.png)
```
Comment (slackish) [inline site/index.md:16]:
```suggestion
Insert text here!
![https://meetup.com/make-roanoke](/assets/images/20241017_launch_party.png)
```

Comment (slackish) [reply site/index.md:16]: Created for a demonstration.  Not needed

File: site/index.md [hunk 2]
```diff
@@ -9,9 +9,11 @@ layout: home
 #  a makerspace in Roanoke, VA.
 ---
 
+### MAKE Roanoke is launching a Kickstarter campaign to open a Makerspace in Roanoke, VA
+
```
Comment (ecks) [inline site/index.md:13]:
```suggestion
![](/assets/images/Kickstarter.png)
```

File: site/index.md (hunk 1 above)
Comment (ecks) [inline site/index.md:16]:
```suggestion
```

**Review by ecks — CHANGES_REQUESTED** [review]

File: site/index.md (hunk 2 above)
Comment (slackish) [reply site/index.md:13]: We are no longer using that, and replacing this picture with the kickstarter party image:

https://discord.com/channels/1179210982258647100/1287907844695396385/1296583231554060348

File: site/index.md (hunk 1 above)
Comment (slackish) [reply site/index.md:16]: We are using the kickstarter launch party image

https://discord.com/channels/1179210982258647100/1287907844695396385/1296583231554060348

**Review by ecks — APPROVED** [review]

File: site/index.md [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
+image: /assets/images/20241017_kickstarter_website_banner.png
 layout: home
 #introduction: | 
 #
 #  You've found the site for MAKE Roanoke. We are a non-profit building
 #  a makerspace in Roanoke, VA.
 ---
 
+### MAKE Roanoke is launching a Kickstarter campaign to open a Makerspace in Roanoke, VA
+![](/assets/images/Kickstarter.png)
```
Comment (ecks) [inline site/index.md:13]:
```suggestion
```

**Pull request merged**